src/AzureAuth/Commands/Ado/CommandPat.cs

```diff
@@ -200,8 +203,12 @@ private bool ValidOptions(ILogger logger)
 
         private TokenResult AccessToken(IPublicClientAuth publicClientAuth, CommandExecuteEventData eventData)
         {
+            var authParams = new AuthParameters(

```

Comment:
Totally unrelated to this PR, but related to the previous one. Sorry I just thought about it - we have two constants file one of which is introduced in the previous PR - https://github.com/AzureAD/microsoft-authentication-cli/blob/main/src/AzureAuth/Ado/Constants.cs  and this one https://github.com/AzureAD/microsoft-authentication-cli/blob/main/src/AdoPat/Constants.cs 

Do we need both the files or can we merge them?

Reply:
I'm not sure it's prudent to merge them without merging the rest. The constants in `AzureAuth.Ado.Constants` are ones which are important for the CLI. The ones in `AdoPat.Constants` are ones which are important for the ADO PAT library (which is used internally by the CLI). There's a subtle, but important difference there, I think. To me, it makes sense to keep them separate for now.